rai/node/bootstrap.cpp

```diff
@@ -1395,6 +1427,11 @@ void rai::bootstrap_server::run_next ()
 	requests.front ()->visit (visitor);
 }
 
+/*
```

Comment:
Doxygen has been introduced. Would be great if these comments started with

    /**